changelog.txt

```diff
@@ -35,6 +35,11 @@ Fixed
 
  - Attempting to create a linked view for a Project on Windows now raises an informative error message.
 
+Deprecated
+++++++++++
+
+ - Modifying the project config, to be removed in 2.0 (#246).
```

Comment:
```suggestion
 - In-memory modification of the project configuration, to be removed in 2.0 (#246).
```
Otherwise it's a bit too vague and might imply that you would not be able to change the project configuration at all.